### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by guewen**
Details in the commits

### CODE REVIEW

File: shopfloor/services/single_pack_transfer.py [hunk 1]
```diff
@@ -204,7 +204,9 @@ def _router_validate_success(self, package_level):
         return self._response_for_start(message=message, popup=completion_info_popup)
 
     def _set_destination_and_done(self, move, scanned_location):
-        move.move_line_ids[0].location_dest_id = scanned_location.id
+        # when writing the destination on the package level, it writes
```
Comment (simahawk) [inline shopfloor/services/single_pack_transfer.py:207]: good to know :)

File: shopfloor/services/cluster_picking.py [hunk 2]
```diff
@@ -957,6 +957,7 @@ def set_destination_all(self, picking_batch_id, barcode, confirmation=False):
 
     def _unload_write_destination_on_lines(self, lines, location):
         lines.write({"shopfloor_unloaded": True, "location_dest_id": location.id})
+        lines.package_level_id.location_dest_id = location
```
Comment (simahawk) [inline shopfloor/services/cluster_picking.py:960]: no fear of multiple recordset write issue?

**Review by simahawk — APPROVED** [review]
LG

**Review by sebalix — APPROVED** [review]

File: shopfloor/services/cluster_picking.py (hunk 2 above)
Comment (guewen) [reply shopfloor/services/cluster_picking.py:960]: since 13.0, relations are automatically mapped

**Pull request merged**